### PR TITLE
add_urls_and_descriptions

### PR DESCRIPTION
**Pull request by tokybe**
Add summaries and  links.

### CODE REVIEW

File: dicts/setup.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
     version="0.1.0",
```
Comment (foxyblue) [inline dicts/setup.py:5]: You need to bump these version numbers too.

File: connections/setup.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
     version="0.1.2",
     author="Demeter Sztanko",
     author_email="demeter.sztanko@revolut.com",
+    description="Get connected instantly to a predefined set of databases, regardless of their type and nature.",
```
Comment (foxyblue) [inline connections/setup.py:9]: I wouldn't say this is it.
It's more like "A utility to pre-configure connects regardless of type."

File: dicts/setup.py (hunk 1 above)
Comment (tokybe) [reply dicts/setup.py]: Done

File: connections/setup.py (hunk 2 above)
Comment (tokybe) [reply connections/setup.py]: Yes, better. Updated.

**Pull request merged**